### PR TITLE
Fixed DB Load event creator list on certain sites.

### PR DESCRIPTION
**Pull request by tonypartridge**

### CODE REVIEW

File: component/admin/language/en-GB.com_jevents.ini [hunk 1]
```diff
@@ -1435,7 +1435,7 @@ JEV_SHOW_TIMEZONE_DESC="When enabled allows the event editor to specify the time
 JEV_CSV_EXPORT="Export results as CSV file"
 JEV_CSV_EXPORT_DESC="Allows exporting of event data in CSV file for chosen columns"
 
-JEV_CHECK_OVERLAPPING_REPEATS="WARNING: You have created a series of repeating events that overlap each other.  This is probably a mistake please check your event dates and repeat pattern"
+JEV_CHECK_OVERLAPPING_REPEATS="WARNING: You have created a series of repeating events that overlap each other. This is probably a mistake please check your event dates and repeat pattern"
```
Comment (GeraintEdwards) [inline component/admin/language/en-GB.com_jevents.ini]: In standard english typing you always have 2 spaces after a full stop, 1 after a comma ;)

File: component/site/libraries/registry.php [hunk 2]
```diff
@@ -11,132 +11,134 @@
  */
```
Comment (GeraintEdwards) [inline component/site/libraries/registry.php]: Are changes to this file only formatting?  I can't see what has changed very clearly.

File: component/admin/language/en-GB.com_jevents.ini (hunk 1 above)
Comment (tonypartridge) [inline component/admin/language/en-GB.com_jevents.ini]: Really... I didn't know that? Plus it's output as a single ;-)

File: component/site/libraries/registry.php (hunk 2 above)
Comment (tonypartridge) [inline component/site/libraries/registry.php]: Yep just formatting. I've checked local history :-)

**Pull request merged**